jujugui/static/gui/src/app/components/inspector/inspector.js

```diff
@@ -53,6 +53,7 @@ YUI.add('inspector-component', function() {
       setCharm: React.PropTypes.func.isRequired,
       setConfig: React.PropTypes.func.isRequired,
       showActivePlan: React.PropTypes.func.isRequired,
+      showPlans: React.PropTypes.bool.isRequired,
```

Comment:
I am not convinced that passing flags as properties is worth the effort, but since you already did it I will not complain.